config/environment.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
 
-module.exports = function(/* environment, appConfig */) {
-  return { };
+module.exports = function (/* environment, appConfig */) {
```

Comment:
nit: thoughts on the space after a function, can we control this? I read up on it like 2 months ago but have forgotten

Reply:
afaik this is a Prettier setting (in v2 iirc) and it looks like there's not a configuration option to change it—tbqh though, this is one of the ones I don't really have a preference for either way.